### DISCUSSION TITLE
lazy git in background

### DISCUSSION THREAD

**Discussion by prabanjanraja** [LazyVim/LazyVim#1287] [Q&A]
I was trying to force push into a repository but it took a long time 
so I pressed esc and q to close lazyvim, 
I was hoping lazygit would work in the background but it was not.
why?
how can I make it so that it works in background?

**Comment by dpetka2001** [comment]
Only if you put neovim in the background, aka press `<Ctrl-z>` to send a process in the background in Linux. But I really don't know if the process will keep operating or just in a stopped state until you bring it back. If it's in a stopped state, then your other option is to run lazygit as an external process and do your stuff there outside of Neovim. You could also open a terminal in a different tab in Neovim and launch lazygit there and then go back to the other stuff where you have your buffers open. But if you close Neovim, I suspect the same thing will happen. So just launch lazygit in your terminal and do your staff there and on another terminal tab do your Neovim stuff and even if you close it there won't be any problems. If none of this satisfies you, maybe you would be better opening an issue at lazygit repo to ask for help how you can keep lazygit working in the background, because I'm not aware of such a thing and can't help you with that.
